### PR TITLE
API-10007 Added full name columns for practitioner and practititionerrole

### PR DESCRIPTION
**Pull request by taylorrassmannlibertyits**

### CODE REVIEW

File: db/migration/V20_0__drop_and_create_practitioner.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+GO
+
+CREATE TABLE [App].[Practitioner]
+(
+  [CDWIdNumber] bigint not null,
+  [CDWIdResourceCode] char(1) not null,
+  [FamilyName] varchar(50) null,
+  [GivenName] varchar(50) null,
+  [FullName] varchar(100) null,
+  [NPI] varchar(50) null,
```
Comment (jtaylor-va) [inline db/migration/V20_0__drop_and_create_practitioner.sql:11]: We also expose `Sta3n` in this view. Should it be included as well for completeness? `Sta3n` is a `smallint`.

**Comment by jblefkove-va** [comment]
Since this is a simpler changeset, let's add the individual new column (or two) vs. recreating the whole table

File: db/migration/V20_0__drop_and_create_practitioner.sql (hunk 1 above)
Comment (nmoisan) [reply db/migration/V20_0__drop_and_create_practitioner.sql:11]: I can't remember the full conversation, but I think the end result was the API doesn't need it at this time. I also think I will be removing it when I deploy the new `Practitioner` so I thin we can leave it out.

**Review by jtaylor-va — APPROVED** [review]
LGTM!

**Review by nmoisan — APPROVED** [review]

**Review by jblefkove-va — APPROVED** [review]
:shipit:

**Pull request merged**